Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,32 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
+  homepage "https://sites.google.com/site/gogdownloader"
+  url "https://sites.google.com/site/gogdownloader/lgogdownloader-3.2.tar.gz"
+  sha256 "9e60af50c18533884ca317778b291152d5f04d93dfd3dc2579f6f965538b2031"
+
+  depends_on "boost"
+  depends_on "htmlcxx"
+  depends_on "jsoncpp"
+  depends_on "liboauth"
+  depends_on "rhash"
+  depends_on "tinyxml2"
+
+  depends_on "cmake" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    mkdir "build" do
+      system "cmake", "..",
```

Comment:
See other formulas for how we indent this

Reply:
Can you please elaborate? I feel the level of indentation is the same as all the other formulas that use the `mkdir` block (four spaces before `mkdir`, six spaces before `system`).

Example: `aces_container.rb`

```
  def install
    mkdir "build" do
      system "cmake", "..", *std_cmake_args
      system "make", "install"
    end
  end
```

Reply:
I meant for arguments to `cmake`. We usually fit more that one per line, and align next lines on the first argument.

Reply:
Fixed.